### ISSUE TITLE
go through README's tutorial, john account can't view space in example way

### ISSUE THREAD

**Issue by xiaods** [kiosk-sh/kiosk#48]
Environment:
mac
docker desktop
kubernetes 1.16.5

I follow README's tutorial, when I step to create space steps,  I came across bug here:
```
~ took 15s 
❯ kubectl apply -f https://raw.githubusercontent.com/kiosk-sh/kiosk/master/examples/space.yaml --as=john
space.tenancy.kiosk.sh/johns-space created

~ took 35s 
❯ kubectl get spaces --as=john
No resources found.

~ took 12s 
❯ kubectl get spaces --as=john
No resources found.

~ took 3s 
❯ kubectl get space johns-space -o yaml --as=john
Error from server (NotFound): space.tenancy.kiosk.sh "johns-space" not found
```

how to debug this situation?

**Comment by LukasGentele** [comment]
@xiaods Thanks for reporting this.

Did you execute this command beforehand? 
```bash
kubectl apply -f https://raw.githubusercontent.com/kiosk-sh/kiosk/master/examples/rbac-viewer.yaml
```
It is shown in [step 2.2](https://github.com/kiosk-sh/kiosk#22-allow-users-to-view-accounts) within the readme.

**Comment by xiaods** [comment]
@LukasGentele I confirm I already execute this command.

```
❯ kubectl get accounts --as=john
NAME            AGE
johns-account   5m28s

~/Desktop took 31s 
❯ kubectl get account johns-account -o yaml --as=john
apiVersion: tenancy.kiosk.sh/v1alpha1
kind: Account
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"tenancy.kiosk.sh/v1alpha1","kind":"Account","metadata":{"annotations":{},"name":"johns-account"},"spec":{"subjects":[{"apiGroup":"rbac.authorization.k8s.io","kind":"User","name":"john"}]}}
  creationTimestamp: "2020-04-08T02:21:37Z"
  generation: 1
  name: johns-account
  resourceVersion: "16494"
  selfLink: /apis/tenancy.kiosk.sh/v1alpha1/accounts/johns-account
  uid: b3f4cffe-8305-4f7a-9a58-1895816801fc
spec:
  space:
    spaceTemplate:
      metadata:
        creationTimestamp: null
  subjects:
  - apiGroup: rbac.authorization.k8s.io
    kind: User
    name: john
status:
  namespaces:
  - name: johns-space
```

**Comment by xiaods** [comment]
it seems a bug on it.

**Comment by xiaods** [comment]
potential issue:

```
Events:
  Type     Reason       Age                    From                     Message
  ----     ------       ----                   ----                     -------
  Normal   Scheduled    8m41s                  default-scheduler        Successfully assigned kiosk/kiosk-7bdbb6d9cd-z4js5 to docker-desktop
  Warning  FailedMount  8m41s                  kubelet, docker-desktop  MountVolume.SetUp failed for volume "cert" : secret "webhook-server-cert" not found
  Warning  FailedMount  8m41s (x2 over 8m41s)  kubelet, docker-desktop  MountVolume.SetUp failed for volume "cert-apiservice" : secret "webhook-server-cert-apiservice" not found
  Normal   Pulled       8m39s                  kubelet, docker-desktop  Container image "kiosksh/kiosk:0.1.3" already present on machine


```

**Comment by xiaods** [comment]
@LukasGentele I confirm this is reproduced bug on docker - mac desktop

**Comment by LukasGentele** [comment]
Thanks for reporting this bug and for all the detailed information @xiaods - We just made some changes regarding listing accounts and spaces, so this seems to be a bug. I told @FabianKramm about this and he will fix this as soon as possible.

**Comment by xiaods** [comment]
@LukasGentele Thanks for your helps.

**Comment by FabianKramm** [comment]
@xiaods thanks for reporting! This is fixed with v0.1.4
